<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <rosparam command="load" file="/home/meguken/catkin_ws/src/imu_gnss_localizer/conf/conf.yaml"/>

  <node pkg="imu_gnss_localizer" name="RCalc_VelocitySF" type="RCalc_VelocitySF" />
  <node pkg="imu_gnss_localizer" name="RCalc_YawrateOffsetStop" type="RCalc_YawrateOffsetStop" />
  <node pkg="imu_gnss_localizer" name="RCalc_YawrateOffset1st" type="RCalc_YawrateOffset" args="1st"/>
  <node pkg="imu_gnss_localizer" name="RCalc_YawrateOffset2nd" type="RCalc_YawrateOffset" args="2nd"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading1st" type="RCalc_Heading" args="1st"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading2nd" type="RCalc_Heading" args="2nd"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading3rd" type="RCalc_Heading" args="3rd"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading1st_Int" type="RCalc_Heading_Int" args="1st"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading2nd_Int" type="RCalc_Heading_Int" args="2nd"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Heading3rd_Int" type="RCalc_Heading_Int" args="3rd"/>
  <node pkg="imu_gnss_localizer" name="RCalc_Distance" type="RCalc_Distance" />
  <node pkg="imu_gnss_localizer" name="RCalc_Trajectory" type="RCalc_Trajectory" />
  <node pkg="imu_gnss_localizer" name="RCalc_PositionDis" type="RCalc_PositionDis" />
  <node pkg="imu_gnss_localizer" name="RCalc_PositionDis_Int" type="RCalc_PositionDis_Int" />
  <node pkg="imu_gnss_localizer" name="Debug_tool" type="Debug_tool" />

  <param name="/imu_gnss_localizer/reverse_imu" type="bool" value="false" />

  <node pkg="toyota_can" name="toyota_can" type="toyota_can" />

</launch>
